lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.SpaceType.rb

```diff
@@ -26,28 +26,8 @@ def whole_building_space_type_name(model, primary_building_type)
   #        define (ventilation, exhaust, lighting control) from ASHRAE9012007
   # @return [hash] hash of internal loads for different load types
   def space_type_get_standards_data(space_type, extend_with_2007: true, throw_if_not_found: false)
-    # This returns 'Office' for eg
-    standards_building_type = model_get_primary_building_type(space_type.model, remap_office: false)
 
-    # populate search hash
-    search_criteria = {
-      'template' => template,
-      'building_type' => standards_building_type,
-      'space_type' => whole_building_space_type_name(space_type.model, standards_building_type),
-    }
-
-    # lookup space type properties
-    space_type_properties = model_find_object(standards_data['space_types'], search_criteria)
-
-    if space_type_properties.nil?
-      msg = "Space type properties lookup failed: #{search_criteria}."
-      if throw_if_not_found
-        OpenStudio.logFree(OpenStudio::Error, 'openstudio.standards.SpaceType', msg)
-        raise msg
-      end
-      OpenStudio.logFree(OpenStudio::Warn, 'openstudio.standards.SpaceType', msg)
-      space_type_properties = {}
-    end
+    space_type_properties = model_get_standards_data(space_type.model, throw_if_not_found: throw_if_not_found)
```

Comment:
space_type_get_standards_data will use that new helper, and extend_with_2007 if requested.